### DISCUSSION TITLE
戳一戳 判断不出来

### DISCUSSION THREAD

**Discussion by Xiaoprinceprogrammer** [Mrs4s/go-cqhttp#1614] [Q&A] [answered]
{"post_type":"message","message_type":"private","time":1659595469,"self_id":2401858097,"sub_type":"friend","user_id":1317967909,"target_id":2401858097,"message":"","raw_message":"","font":0,"sender":{"age":0,"nickname":"晓王子~程序员","sex":"unknown","user_id":1317967909},"message_id":-1505963132}
”你们看这是抖一抖的数据，哪有你们写的参数？

**Comment by Mrs4s** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/12137562/182783391-14cce393-f703-4eaf-b094-38e9b8779120.png)
能判断

**Reply by Xiaoprinceprogrammer** [reply]
我为什么判断不出来
![QQ截图20220804150048](https://user-images.githubusercontent.com/110460831/182784179-255aea75-6764-42c1-a573-0f2453609eae.jpg)
![QQ截图20220804150141](https://user-images.githubusercontent.com/110460831/182784183-0ab7ff11-1bc9-4462-9d03-be198bff93da.jpg)

**Reply by Ink-33** [reply]
> 我为什么判断不出来
> ![QQ截图20220804150048](https://user-images.githubusercontent.com/110460831/182784179-255aea75-6764-42c1-a573-0f2453609eae.jpg)
> ![QQ截图20220804150141](https://user-images.githubusercontent.com/110460831/182784183-0ab7ff11-1bc9-4462-9d03-be198bff93da.jpg)
> 

请更新最新版本

**Reply by Xiaoprinceprogrammer** [reply]
我抖一抖就是空数据，还有不只是我判断不了好多人也一样

**Reply by Xiaoprinceprogrammer** [reply]
我用的就是最新版本

**Reply by Mrs4s** [reply]
抖一抖不支持 只支持判断戳一戳

**Reply by Xiaoprinceprogrammer** [reply]
![QQ截图20220804150634](https://user-images.githubusercontent.com/110460831/182785160-bcb9a2df-d133-4d64-842c-c8c0ce5cdaf4.jpg)
这个不支持吗

**Reply by Mrs4s** [reply]
不支持

**Reply by Xiaoprinceprogrammer** [reply]
那你们 给你还是不够完善啊

**Reply by Mrs4s** [reply]
键盘给你你来写
有点气人 block了

**Reply by LXY1226** [reply]
键盘给你你来写

**Reply by wyapx** [reply]
> ![QQ截图20220804150634](https://user-images.githubusercontent.com/110460831/182785160-bcb9a2df-d133-4d64-842c-c8c0ce5cdaf4.jpg) 这个不支持吗

我觉得你可以悬停在这个图标上面
应该会有“窗口抖动”字样

然后去百度查一下窗口抖动和戳一戳的区别

**Comment by Ink-33** [comment]
![image](https://user-images.githubusercontent.com/51873347/182783524-353b205b-ff4b-40c9-a7f6-c5cf8a0569ce.png)
能判断

**Comment by Pandaft** [comment]
这位兄弟注意一下自己的行为，实在是过于经典了，在这罗列一下你这两天都干了啥事：

1. （[传送门](https://github.com/Mrs4s/go-cqhttp/pull/1611)）通过 PR 向开发者请求添加新功能，你应该提 issue 或 discussion 而不是 PR ，并且要礼貌。
2. （[传送门](https://github.com/Mrs4s/go-cqhttp/discussions/1612#discussioncomment-3314561)）图片上传成功没自己不会 preview 一下吗，发表之后自己不会打开看看吗。
3. （[传送门](https://github.com/Mrs4s/go-cqhttp/discussions/1612#discussioncomment-3314600)）对 arm64 、armv7 等架构词不了解，为何不试试谷歌或百度？
4. （[传送门](https://github.com/Mrs4s/go-cqhttp/discussions/1614#discussioncomment-3322713)）此 discussion 中分不清 `抖一抖` 和 `戳一戳` ，并且指责项目不完善。

如果你不会 GitHub 的 PR、issue、discussions 等功能就悠着点，摸索会了再用。
尊重项目贡献者们的无偿奉献，此项目的 README 中写着：

> 请注意, 开发者并没有义务回复您的问题. 您应该具备基本的提问技巧。
> 有关如何提问，请阅读[《提问的智慧》](https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md)

实在不行就去读一遍吧。
